docs/kyma-release-notes/2023-10-02-release-notes-2.19.md

```diff
@@ -0,0 +1,35 @@
+
+## Application Connectivity
+
+Additional logging options have been added to Central Application Gateway.
+https://github.com/kyma-project/kyma/#debugging
+
+## Observability
+
+Early warning: The Monitoring component will be removed with Kyma 2.20, as announced in this [blog post](https://blogs.sap.com/2023/09/07/removal-of-prometheus-grafana-based-monitoring-in-sap-btp-kyma-runtime/).
```

Comment:
Maybe this could be a tab ? It would be more eye-catching.

Reply:
why not, cannot imagine how it will look like, so can you do the change and then let's see?

Reply:
I meant a panel, of course :D